microsoft-edge/progressive-web-apps/samples/temperature-converter.md

```diff
@@ -0,0 +1,804 @@
+
+1. During installation, you can accept the defaults, except it's recommended that you change the default editor from vim to Visual Studio Code:
+
+   ![git installer with Visual Studio Code selected as default editor](./temperature-converter-images/git-default-editor.png)
+
+
+<!-- ====================================================================== -->
+## Step 4: Install Node.js
+
+Node.js includes a web server that you'll use to run and test the sample locally.
+
+Install or update Node.js, as follows:
+
+1. Open Visual Studio Code.
+
+1. In the **View** menu, select **Terminal**.
+
+1. Enter the command: `node -v`
+
+   If Node.js has been installed, a version number is output, such as v22.19.0.
+
+1. Go to [Node.js](https://nodejs.org).
+
+1. In the lower left, in the **Latest LTS** button, note the version number, such as v22.20.0.
+
+1. If Node.js isn't installed yet, or you want to update it, click the **Get Node.js** button.  Otherwise, skip to the next section, below.
+
+   You end up at a page such as [Download Node.js](https://nodejs.org/download).
+
+1. Click the **Windows Installer (.msi)** button.
+
+   The **Downloads** window of Microsoft Edge opens, and displays a file name such as `node-v22.20.0-x64.msi`.
+
+1. Click the **open file** link under the file name.
+
+   The **Node.js Setup Wizard** window opens.
+
+1. Click the **Next** button, and follow the prompts.  You can accept the defaults, and then click the **Install** button.
+
+   The **Node.js Setup Wizard** window closes.
+
+1. In VS Code's terminal, enter the command: `node -v`
+
+   The latest version number is displayed, such as v22.20.0.
+
+
+<!-- ====================================================================== -->
+## Step 5: Fork and clone the "MicrosoftEdge / Demos" repo
+<!-- compare: https://learn.microsoft.com/microsoft-edge/devtools/sample-code/sample-code#clone-the-edge-demos-repo-to-your-drive -->
+
+Next, you'll get a local copy of the sample code, by forking the Microsoft Edge / Demos repo.  Because you don't have permissions to directly push commits to the Demos open-source repo, you'll _fork_ the Demos repo to create your own copy of the repo, instead of _cloning_ the Demos repo.
+
+You might be able to download the "main" branch of the Demos repo, and edit the files in the "main" branch, rather than forking and cloning the repo and then creating a working branch (test1) in which to edit the files.  This article shows the more complex but flexible way: forking the repo.
+
+Fork the **MicrosoftEdge / Demos** repo, as follows:
+
+1. Press **Windows+E**.
+
+   File Explorer opens.
+
+   **Check whether Demos repo has been cloned:**
+
+1. Navigate to where you want the repo cloned to, such as:
+
+   `C:\Users\localAccount\GitHub\`
+
+1. See whether the `\Demos\` repo is listed, such as `C:\Users\localAccount\GitHub\Demos`.
```

Comment:
If the person is going through these steps, there's very little chances that they'll already have cloned the Demos repo. I think this step can safely be skipped.